src/main/java/com/github/fge/jsonschema/main/JsonSchemaFactory.java

```diff
@@ -60,7 +60,7 @@
  *     <li>a {@link SyntaxValidator}, to validate schemas;</li>
  *     <li>a {@link JsonValidator}, to validate an instance against a schema;
  *     </li>
- *     <li>a {@link JsonSchema}, to validate instances against a fixed schema.
+ *     <li>a {@link JsonSchemaImpl}, to validate instances against a fixed schema.
```

Comment:
Oh no, I'm good with the patchset overall.   I was asking this specific line it seems like ti would make sense pointing at the interface.